src/vendor/react-dom/reactsGetEventKey.js

```diff
@@ -28,50 +29,6 @@ const normalizeKey = {
   MozPrintableKey: 'Unidentified',
 };
 
-/**
```

Comment:
🐥☂️Abstract React's list of non-printable keys into a separate file, so it can be re-used by `react-hotkeys`.